### DISCUSSION TITLE
Discrepancy between VEP and Alamut - deletion spanning stop codon, fs or ext?

### DISCUSSION THREAD

**Discussion by WhispAllium** [HGVSnomenclature/hgvs-nomenclature#177] [Other discussion topics]
The c.7917_7918del variant in FLNA gene spans the last nucleotide before the stop codon and the first nucleotide of the stop codon. The resulting frameshift creates a change that does not impact the predicted protein of the last coding amino acid, but does replace the stop codon with an extension. 
NM_001456.4:g.153577219_153577220del

Alamut provides this protein change
p.(*2640Serext*100)

VEP provides this protein change
NP_001447.2:p.Ter2640SerfsTer101

One is ext and one is fs. Which is correct? HGVS doesn’t seem to have this exact scenario covered.

**Comment by jtdendunnen** [comment]
Just to be sure, I assume you mean variant NC_000023.10:g.153577219_153577220del NM_001456.3:c.7917_7918del.

The predicted consequence on protein level of this variant is indeed an extension ("a sequence change extending the reference amino acid sequence at the N- or C-terminal end with one or more amino acids"), following HGVS nomenclature described as NP_001447.2:p.(Ter2640SerextTer100). For details see https://hgvs-nomenclature.org/stable/recommendations/protein/extension/.
